support-frontend/assets/components/thankYou/supportReminder/supportReminderItems.tsx

```diff
@@ -172,14 +169,9 @@ export function SupportReminderBodyCopy({
 									value={choice.label}
 									label={choice.label}
 									checked={supportReminderState.selectedChoiceIndex === index}
-									onChange={() =>
-										dispatch(
-											setThankYouSupportReminder({
-												...supportReminderState,
-												selectedChoiceIndex: index,
-											}),
-										)
-									}
+									onChange={() => {
+										onChange ? onChange(index) : undefined;
```

Comment:
No fuss, but as we've done it elsewhere
```suggestion
										onChange?.(index);
```

Reply:
Ah missed that one! Ta, Updated